csharp/src/Ice/InputStream.cs

```diff
@@ -2588,63 +2549,58 @@ private AnyClass ReadInstance(int index)
             }
 
             // Read the instance.
-            v.iceRead(this);
+            v.Read(this);
```

Comment:
Just call `v.IceRead(this, true)` if we get rid of `Read`

Reply:
See IceRead comment above.